src/app/app.component.html

```diff
@@ -286,8 +287,15 @@
                           <!-- Responses dropdown -->
                           <div *ngIf="activeRouteResponse$ | async as activeRouteResponse" class="btn-group dropdown mr-1" ngbDropdown>
                             <button type="button" class="btn btn-custom" (click)="addRouteResponse()">
-                              <i class="material-icons">add</i> <span class="d-none d-xl-inline">Add response</span>
+                              <i class="material-icons" ngbTooltip="Add response">add</i>
+                            </button>
+
+                            <button type="button" class="btn btn-custom" (click)="toggleRandomResponse(activeRoute.uuid)">
+                              <i
+                                [ngClass]="{'text-primary': getRandomResponse()}"
```

Comment:
you should directly use `activeRoute.randomResponse` which is available there through an `async` pipe line 238:

```suggestion
                                [ngClass]="{'text-primary': activeRoute.randomResponse}"
```